parsers/IEMOP.py

```diff
@@ -477,10 +480,14 @@ def get_all_market_reports_items(
         "page": "1",
         "post_id": KIND_TO_POST_ID[kind],
     }
-    r = requests.post(
-        "https://www.iemop.ph/wp-admin/admin-ajax.php", data=form_data, verify=False
-    )
-    id_to_items = eval(r.text)["data"]
```

Comment:
I don't know how this got accepted before - but using `eval` is a very bad practice, especially if it's just `json` and we can run `json.loads`

Reply:
If it's actually JSON we can just do r.json() and then grab the data dict/list.